guides/common/modules/proc_upgrading-a-disconnected-project-server.adoc

```diff
@@ -73,40 +73,11 @@ Reboot these hosts after the upgrade has completed.
 # rm /etc/yum.repos.d/*
 ----
 
-. Obtain the latest ISO files by following the {InstallingServerDisconnectedDocURL}downloading-the-binary-dvd-images_satellite[Downloading the Binary DVD Images] procedure in _{InstallingServerDisconnectedDocTitle}_.
+. Create a syncable format export for the {Project} tools and {Project} maintenance repositories. 
```

Comment:
I'm missing more specific instructions here. How do I create the export? This looks like a good place for an actual command.

Reply:
Alternative idea: link to https://docs.theforeman.org/nightly/Managing_Content/index-katello.html#Exporting_a_Repository_in_a_Syncable_Format_content-management